src/app/components/SubjectHeading/SubjectHeadingsTableBody.jsx

```diff
@@ -68,13 +70,31 @@ class SubjectHeadingsTableBody extends React.Component {
     this.setState(prevState => prevState);
   }
 
+  subHeadingHeadings() {
+    if (this.props.top) return [];
+    return [
+      {
+        label: 'Heading',
+        labelSorter: this.props.labelSorter,
+        uuid: 'blah',
```

Comment:
This seems to be causing a bug, because "Heading" in the subtable header becomes a link to subject heading with UUID "blah"